lib/api/element-commands.js

```diff
@@ -341,10 +342,36 @@ module.exports = function(client) {
     events.EventEmitter.call(this);
 
     var $this = this;
-    var el = Protocol.element(using, value, function(result) {
+
+    // element commands support a single element but we use elements() to capture
```

Comment:
I noticed you have a preference for detailed commenting. How strongly do you feel about this? So far throughout the codebase I followed a policy of fewer inline comments in favour of more self explanatory methods/functions and if necessary move the comment there. I believe this makes the code easier to follow.